tools/options.c

```diff
@@ -355,6 +357,7 @@ static void nn_append_string_to_free (struct nn_parse_context *ctx,
 
     lst = (struct nn_string_list *)(
         ((char *)ctx->target) + opt->offset);
+    nn_assert (lst);
```

Comment:
As above, probably better to assert the ctx rather than lst.  The compiler should be able to figure it out from there.